internal/provider/schema_gitlab_group.go

```diff
@@ -0,0 +1,162 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func gitlabGroupSchema() map[string]*schema.Schema {
```

Comment:
❓ **Question:**  what do you think about making use of this in the other group resources / data sources, too?

Reply:
I think it makes sense, but we should do that on a different PR, so we don't increase the scope of this one. wdyt?

Reply:
@timofurrer